.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,117 @@
+name: 🐛 Bug Report
+description: Help us improve this repository by filing a detailed bug report.
+body:
+  - type: input
+    id: nodeVersion
+    attributes:
+      label: Node Version
+      description: The version of Node.js you are using (only LTS versions are supported).
+      placeholder: 20.0.0
+    validations:
+      required: true
+
+  - type: input
+    id: pmVersion
+    attributes:
+      label: NPM/Yarn/PNPM Version
+      description: The package manager and version you are using.
+      placeholder: NPM 10.0.0
+    validations:
+      required: true
+
+  - type: input
+    id: strapiVersion
+    attributes:
+      label: Strapi Version
+      description: The version of Strapi you are using ("Latest" is not a valid response).
+      placeholder: 5.7.0
+    validations:
+      required: true
+
+  - type: dropdown
+    id: os
+    attributes:
+      label: Operating System
+      description: Select the operating system where you are experiencing the issue.
+      multiple: false
+      options:
+        - Strapi Cloud
+        - MacOS
+        - Linux (Debian/Ubuntu)
+        - Linux (RedHat/CentOS)
+        - Linux (Other)
+        - Windows 10
+        - Windows 11
+        - Docker/Podman/LXC
+        - Other
+    validations:
+      required: true
+
+  - type: input
+    id: url
+    attributes:
+      label: Reproduction URL
+      description: If you would like to provide a link to a repository that reproduces the issue, please provide it here.
+      placeholder: https://github.com/strapi/LaunchPad
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Bug Description
+      description: Provide a clear and concise description of the issue or unexpected behavior.
+      placeholder: When fetching data from the REST API, I see a HTTP 500 error response.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Steps to Reproduce
+      description: Provide a detailed, step-by-step guide on how to reproduce the issue. Include code snippets or links to repositories if applicable.
+      placeholder: |
+        Example:
+          1. Create a content type in Strapi with fields: title, content.
+          2. Add test data to the new content type
+          3. Perform a REST API call to fetch the data
+          4. Observe the unexpected behavior or error in the response/output.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: Provide a detailed explanation of what you expected to happen instead of the observed issue. Be as specific as possible.
+    validations:
+      required: true
+
+  - type: textarea
+    id: logs
+    attributes:
+      label: Logs
+      description: If available, please provide the logs generated either in the Browser or Server Side console.
+      render: shell
```

Comment:
Considering there might be more than one log, I wonder if we might want a text area for "additional information" that can be used for whatever the reporter feels like adding

Reply:
added another block for that, I added the logs one because it has a special renderer I want to take a look at